modules/cassandra/src/main/scala/sup/cassandra.scala

```diff
@@ -0,0 +1,18 @@
+package sup
+
+import cats.Id
+import cats.effect.implicits.catsEffectSyntaxConcurrent
+import cats.effect.{Concurrent, Sync, Timer}
+import com.datastax.oss.driver.api.core.CqlSession
+
+import scala.concurrent.duration.FiniteDuration
+
+object cassandra {
+
+  def connectionCheck[F[_] : Sync : Concurrent](session: CqlSession, timeout: FiniteDuration)(implicit timer: Timer[F]): HealthCheck[F, Id] =
+    HealthCheck.liftFBoolean {
+      Sync[F].delay(session
+        .execute("SELECT now() FROM system.local").isFullyFetched
+      ).timeoutTo(timeout, Sync[F].pure(false))
```

Comment:
We can reuse this mod here: https://sup.kubukoz.com/guide/#timeouttosick

Reply:
I changed the way I did it. It'll be the client who will use https://sup.kubukoz.com/guide/#timeouttosick

Reply:
Cool :)